apps/gallery/test/marionette/fullscreen_image_test.js

```diff
@@ -41,57 +41,59 @@ marionette('the gallery', function() {
   test('should display an image fullscreen and go back', function() {
     // You should be able to click on an image to view a fullscreen
     // preview and go back by pressing the 'back' button.
-    app.thumbnail.click();
-    assert.ok(app.fullscreenView.displayed());
+    fullscreen_view = app.tapFirstThumbnail();
+    assert.ok(fullscreen_view.displayed);
 
-    app.fullscreenBackButton.click();
+    fullscreen_view.fullscreenBackButton.click();
     assert.ok(app.thumbnailsView.displayed());
   });
 
   test('should flick through images in fullscreen mode', function() {
     // Acquire a duplicate of an image by launching the editing
     // mode and saving it.
-    app.thumbnail.click();
+    fullscreen_view = app.tapFirstThumbnail();
     client.waitFor(function() {
-      return app.fullscreenView.displayed();
+      return fullscreen_view.displayed;
     });
-    app.editButton.click();
-    app.waitForImageEditor();
-    app.waitForAutoEnhanceButtonOff();
-    app.editEnhanceButton.click();
-    app.waitForAutoEnhanceButtonOn();
+    fullscreen_view.editButton.click();
+    fullscreen_view.waitForImageEditor();
+    fullscreen_view.waitForAutoEnhanceButtonOff();
+    fullscreen_view.editEnhanceButton.click();
+    fullscreen_view.waitForAutoEnhanceButtonOn();
     client.waitFor(function() {
-      return app.editSaveButton.enabled();
+      return fullscreen_view.editSaveButton.enabled();
     });
-    app.editSaveButton.click();
+    fullscreen_view.editSaveButton.click();
     client.waitFor(function() {
       return app.thumbnails.length == 2;
     });
-    assert.ok(app.fullscreenView.displayed());    
+    assert.ok(fullscreen_view.displayed);    
     
     // go back to the thumbnail view, and click the first thumbnail
-    app.fullscreenBackButton.click();
+    fullscreen_view.fullscreenBackButton.click();
     client.waitFor(function() {
       return app.thumbnailsView.displayed();
     });
-    app.thumbnail.click();
+    app.tapFirstThumbnail();
```

Comment:
the method was already there, so I used it instead rather than using the click() method, to show that the first thumbnail is being clicked.